mock/editors/tools.md

```diff
@@ -0,0 +1,26 @@
+# Document Creation and Tools
+## Menu
+* [Style for Group-internal Drafts](#) (below)
+* [ReSpec and BikeShed](#) (below)
```

Comment:
Consider adding a "legacy tools" section? e.g. for xmlspec.

Reply:
Added.